### PR TITLE
Added Blobtk tool

### PR DESCRIPTION
**Pull request by SaimMomin12**
This PR relates to adding of a new tool **blobtk** to Galaxy. 

This tool uses the Blob directory files and creates static plots for assembly QC.

### CODE REVIEW

**Comment by bgruening** [comment]
Do you want to integrate this one in the other tool? The merged container thing? Sorry, lost track here.

File: tools/blobtk/blobtk.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<tool id="blobtk_plot" name="Blobtk Plot" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@" license="MIT">
+    <description>Processes a BlobDir and produces static plots</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+        mkdir -p blobtk_files && 
+        tar -xvf "$blobdir_input" -C blobtk_files &&
+        blobtk plot --blobdir blobtk_files  --view "$plot_view"
```
Comment (bgruening) [inline tools/blobtk/blobtk.xml:10]: I guess we can integrate this into the other tool, easily.

**Comment by SaimMomin12** [comment]
Yes indeed, I think merging two tools would be a good idea. The docker images for blobtoolkit and blobtk can be found in the links below: 

**blobtoolkit** --> https://hub.docker.com/r/genomehubs/blobtoolkit
**blobtk** --> https://hub.docker.com/r/genomehubs/blobtk

**blobtk** Git Repo --> https://github.com/blobtoolkit/blobtk/wiki

**Comment by bgruening** [comment]
I have put this into draft. I hope that is ok.

**Comment by SaimMomin12** [comment]
The functionality of Blobtk tool was integrated into Blobtoolkit tool as in https://github.com/bgruening/galaxytools/pull/1462

Closing this PR

**Pull request closed without merging**